plugins/opa/pkg/opa/types.go

```diff
@@ -69,12 +68,6 @@ func (o *OutputFormat) SetDefaults(others ...OutputFormat) {
 	}
 }
 
-// CustomCheck is a custom OPA check.
-type CustomCheck struct {
-	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
-	Spec              CustomCheckSpec
-}
-
 // CustomCheckSpec is the body of a Custom Check object
 type CustomCheckSpec struct {
```

Comment:
I think we can remove this one too now, right?

Reply:
correct. Done